packages/mco/components/create-dr-policy/create-dr-policy.tsx

```diff
@@ -200,8 +178,8 @@ export const CreateDRPolicy: React.FC<{}> = () => {
 
   const areDRPolicyInputsValid = () =>
     !!state.policyName &&
-    Object.keys(state.selectedClusters)?.length === MAX_ALLOWED_CLUSTERS &&
-    state.isODFDetected;
+    !!state.replicationType &&
+    Object.keys(state.selectedClusters)?.length === MAX_ALLOWED_CLUSTERS;
```

Comment:
```suggestion
    Object.keys(state.selectedClusters).length === MAX_ALLOWED_CLUSTERS;
```
`Object.keys` will throw error for null values.

Reply:
state.selectedClusters is already a list, i dont know why I added Object.keys(state.selectedClusters) for a list :)